include/evmc/utils.h

```diff
@@ -0,0 +1,19 @@
+/* EVMC: Ethereum Client-VM Connector API.
+ * Copyright 2018 Pawel Bylica.
+ * Licensed under the MIT License. See the LICENSE file.
+ */
+
+#pragma once
+
+#if __cplusplus
+#define EVMC_EXTERNC extern "C"
+#else
+#define EVMC_EXTERNC
+#endif
+
+#if _MSC_VER
+#define EVMC_ENTRYPOINT(function_name)
```

Comment:
@gumb0 do you want to play with this on Windows? 
See https://stackoverflow.com/questions/2290587/gcc-style-weak-linking-in-visual-studio